### DISCUSSION TITLE
SMTP server problem

### DISCUSSION THREAD

**Discussion by vitorcherulli** [chatwoot/chatwoot#9267] [Q&A]
Hello everything is fine?

I have a ChatWoot installation using Portainer, and I would like to know how I can change the SMTP server?

In the initial installation we used Brevo, but now I have a new SMTP.

Could anyone guide me which file I should change with the new information through Portainer?

I'm self hosting, and the magic link itself doesn't work. The login email does not arrive!

**Comment by zz** [comment]
Yes. Chatwoot is a very bad project :(
